Adds new loadout items.

TheStryker opened the pull request:
Adding some items to the loadout, salvaged from Mwahahaha's PR (https://github.com/Aurorastation/Aurora.3/pull/9164) with most of the ports removed.

**Additions:**

- Created and added a colourable, pinnable tag (e.g., those small strips you put your name on and wear)
- Added magenta-blue hanbok and white-pink hanbok.
- Readded female executive suits. (I understand they were previously removed, I'm willing to discuss this.)
- Added short sleeved, crop top, and sleeveless versions of the hoodie.
- Created and added colourable full leather gloves
- Created and added colourable fingerless gloves
- Ported over studded and dangle earrings from Polaris.
- Ported colorable bracelets from Polaris.
- Ported (and made colorable) the keyhole sweater from Polaris.

**Bugs:**
- The earrings are bugged and only have their color applied when something is in the left ear slot.

![loadout PR](https://user-images.githubusercontent.com/19821403/87853955-dd882b00-c940-11ea-828b-acf3db60f32e.PNG)

TheStryker commented:
!review

alsoandanswer commented:
The female executive suits and Hanboks cannot be added.
Please read below.
https://forums.aurorastation.org/topic/14572-policy-update-on-loadout-items/?tab=comments#comment-137818

TheStryker commented:
Hi, sorry for the misunderstanding.
I sprited the hanboks in order to fill a line-up with the cheongsams and kimonos. Furthermore, Konyang has Korean-descended cities and a Korean aesthetic which the hanbok could fit into. (I could also change all references of Korea with Konyang.)
I included the suit mainly because some people expressed an interest in using it again for their characters, but if you want to maintain a distinction between suits, we could perhaps remove the pants suit. As for culture, I thought it fit mainly into generic as a standard business suit.

TheStryker commented:
- Went ahead and changed the references of the mythical land of Korea to Konyang.
- Made a konyang.dm to fit in with categorization.
- Fixed a glitch where the outerwear sprite for hanboks would not load.

Resolved lore issue with hanboks (in description as well)
![PROOF](https://user-images.githubusercontent.com/19821403/88701350-7c303b00-d13c-11ea-8eca-36c214dea9c6.PNG)

TheStryker commented:
Removed the suit, fixed merge conflicts. I apologize for the massive pile of commits.

alsoandanswer commented on code/modules/client/preference_setup/loadout/loadout_uniform.dm:
```diff
@@ -316,4 +316,16 @@
+	gear_tweaks += new/datum/gear_tweak/path(pyjamas)
+
+/datum/gear/uniform/miscellaneous/hanbok
+	display_name = "hanbok"
```
```suggestion
	display_name = "hanbok selection"
```

alsoandanswer commented on code/modules/client/preference_setup/loadout/loadout_suit.dm:
```diff
@@ -64,10 +64,19 @@
 
 /datum/gear/suit/hoodie
-	display_name = "hoodie"
+	display_name = "hoodie (selection)"
```
```suggestion
	display_name = "hoodie selection"
```

alsoandanswer commented on code/modules/clothing/factions/konyang.dm:
```diff
@@ -0,0 +1,17 @@
+	item_state = "hanbok"
+	slot_flags = SLOT_OCLOTHING | SLOT_ICLOTHING
+
+/obj/item/clothing/under/konyang/pink
```
Simce this is the path of the previous hanbok you don't have to copypaste repeated things (e.g. the description)
